### ISSUE TITLE
Documentation lacks using unix socket

### ISSUE THREAD

**Issue by yannfill** [modoboa/modoboa#3241]
# Impacted versions

* OS Type: Arch
* OS Version: current
* Database Type: MySQL
* Database version: 11.3.2
* Modoboa: 2.2.4
* installer used: No
* Webserver: Nginx

The documentation assumes that all connections to external services use TCP, but unix socket may be used too.

I struggled to configure MySQL this way since it's not documented, but now that redis is required, I can't figure out how to use it.

I use this in settings.py:

```
REDIS_HOST = '/run/redis/redis.sock'
REDIS_PORT = 6279
REDIS_QUOTA_DB = 0
REDIS_URL = 'unix://{}?db={}'.format(REDIS_HOST, REDIS_QUOTA_DB)
```

But django seems to keep using REDIS_PORT as i get this error:

```
ConnectionError at /

Error -2 connecting to /run/redis/redis.sock:6279. Name or service not known.
```

Overwriting REDIS_PORT with 0 or empty value keeps throwing errors.

So, how to use redis unix socket?

**Comment by yannfill** [comment]
I finally managed to get it working. The problem was in fact in django-rq configuration. With the modified scheme in `REDIS_URL` I had to add this in the `RQ_QUEUES` config:

```
'URL': REDIS_URL,
```

Also, in the same config, `DB` should reuse `REDIS_QUOTA_DB` instead of hardcoded `0`.

I mentioned struggling with using MySQL socket, so for anyone in the same case, this is the settings to use in `DATABASES` config:

```
'HOST': '/path/to/mysqld.sock',
'PORT': '',
```

And to deploy an instance:

```
(env)$ modoboa-admin.py deploy instance --collectstatic --domain <hostname of your server> --dburl default:mysql://<db_user>:<db_passwd>@localhost/<db_name>
```

I leave this issue opened so that the doc (and the code) will be updated.
